THIRD-PARTY-NOTICES

```diff
@@ -4,7 +4,8 @@ be distributed under licenses different than the MongoDB Java Driver software.
 In the event that we accidentally failed to list a required notice,
 please bring it to our attention through any of the ways detailed here:
 
-           mongodb-dev@googlegroups.com
+To file a bug or feature request: MongoDB Jira Issue Tracker
```

Comment:
This doesn't read properly.  You left a hanging colon.  I think just remove the new sentence about filing a bug or feature request, as it doesn't belong in this file anyway.

Reply:
I have removed the phrase.